src/triage/predictlist/__init__.py

```diff
@@ -0,0 +1,610 @@
+from triage.component.architect.entity_date_table_generators import EntityDateTableGenerator, DEFAULT_ACTIVE_STATE
+from triage.component.architect.features import (
+        FeatureGenerator, 
+        FeatureDictionaryCreator, 
+        FeatureGroupCreator,
+        FeatureGroupMixer,
+)
+from triage.component.architect.feature_group_creator import FeatureGroup
+from triage.component.architect.builders import MatrixBuilder
+from triage.component.architect.planner import Planner
+from triage.component.architect.label_generators import LabelGenerator
+from triage.component.timechop import Timechop
+from triage.component.catwalk.storage import ModelStorageEngine, ProjectStorage
+from triage.component.catwalk import ModelTrainer
+from triage.component.catwalk.model_trainers import flatten_grid_config
+from triage.component.catwalk.predictors import Predictor
+from triage.component.catwalk.utils import retrieve_model_hash_from_id, filename_friendly_hash
+from triage.util.conf import convert_str_to_relativedelta, dt_from_str
+from triage.util.db import scoped_session
+
+from collections import OrderedDict
+import json
+import re
+import random
+from datetime import datetime
+
+import verboselogs, logging
+logger = verboselogs.VerboseLogger(__name__)
+
+
+def experiment_config_from_model_id(db_engine, model_id):
+    """Get original experiment config from model_id 
+    Args:
+            db_engine (sqlalchemy.db.engine)
+            model_id (int) The id of a given model in the database
+
+    Returns: (dict) experiment config
+    """
+    get_experiment_query = '''select experiments.config
+    from triage_metadata.experiments
+    join triage_metadata.models on (experiments.experiment_hash = models.built_by_experiment)
+    where model_id = %s
+    '''
+    (config,) = db_engine.execute(get_experiment_query, model_id).first()
+    return config
+
+
+def experiment_config_from_model_group_id(db_engine, model_group_id):
+    """Get original experiment config from model_id 
+    Args:
+            db_engine (sqlalchemy.db.engine)
+            model_id (int) The id of a given model in the database
+
+    Returns: (dict) experiment config
+    """
+    get_experiment_query = '''
+    select experiment_runs.id as run_id, experiments.config
+    from triage_metadata.experiments
+    join triage_metadata.models 
+    on (experiments.experiment_hash = models.built_by_experiment)
+    join triage_metadata.experiment_runs 
+    on (experiment_runs.experiment_hash = models.built_by_experiment)
+    where model_group_id = %s
+    '''
+    (run_id, config) = db_engine.execute(get_experiment_query, model_group_id).first()
+    return run_id, config
+
+
+def get_model_group_info(db_engine, model_group_id):
+    query = """
+    SELECT m.model_group_id, m.model_type, m.hyperparameters, m.model_id as model_id_last_split
+    FROM triage_metadata.models m
+    JOIN triage_metadata.model_groups mg using (model_group_id)
```

Comment:
Why does this need to join to `model_groups` if everything it pulls is from `models`?